doc/datastorage_setup

```diff
@@ -0,0 +1,36 @@
+Installing ZeroMQ:
+
+# run in sudo
+# Before installing, make sure you have installed all the needed packages
+1.sudo apt-get install libtool pkg-config build-essential autoconf automake
```

Comment:
Revise the instruction applying the coding style font like `sudo apt-get...`.

Reply:
Done